sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/state/MutableMeasurement.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.state;
+
+import io.opentelemetry.api.common.Attributes;
+import javax.annotation.Nullable;
+
+/**
+ * A mutable {@link Measurement} implementation
+ *
+ * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
+ * at any time.
+ *
+ * <p>This class is not thread-safe.
+ */
+public class MutableMeasurement implements Measurement {
+
+  static void setDoubleMeasurement(
+      MutableMeasurement mutableMeasurement,
+      long startEpochNanos,
+      long epochNanos,
+      double value,
+      Attributes attributes) {
+    mutableMeasurement.set(
+        startEpochNanos,
+        epochNanos,
+        /* hasLongValue= */ false,
+        0L,
+        /* hasDoubleValue= */ true,
+        value,
+        attributes);
+  }
+
+  static void setLongMeasurement(
+      MutableMeasurement mutableMeasurement,
+      long startEpochNanos,
+      long epochNanos,
+      long value,
+      Attributes attributes) {
+    mutableMeasurement.set(
+        startEpochNanos,
+        epochNanos,
+        /* hasLongValue= */ true,
+        value,
+        /* hasDoubleValue= */ false,
+        0.0,
+        attributes);
+  }
+
+  private long startEpochNanos;
+  private long epochNanos;
+  private boolean hasLongValue;
+  private long longValue;
+  private boolean hasDoubleValue;
+  private double doubleValue;
+
+  @Nullable private Attributes attributes;
```

Comment:
Is this really nullable?

Reply:
You are correct. I've placed default value of empty attributes and removed the nullable.